python/tvm/relay/tensorrt.py

```diff
@@ -718,7 +749,41 @@ def visit_call(self, call):
         return super().visit_call(call)
 
 
-def PruneSubgraphs(mod, compiler="tensorrt", use_implicit_batch=True, prune_no_macs=False):
+class ComputeHeavyGraph(ExprMutator):
```

Comment:
Should be ExprVisitor since it won't modify. Maybe should rename to `IsComputeHeavyGraph` also

Reply:
Done.